CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2019-12-11
+
+- Functional Options: Recommend individual implementations of `Option`
```

Comment:
I think `Recommend struct implementations` would be more accurate (some options may reuse the same struct)